ccp/modules/exporter-setup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [ -n "$ENABLE_EXPORTER" ];then
```

Comment:
Where is this variable defined? I can't find `ENABLE_EXPORTER` via Ctrl+F

Reply:
This should be defined in the ccp.conf file